mne/io/eeglab/__init__.py

```diff
@@ -4,3 +4,4 @@
 
 from .eeglab import read_raw_eeglab, read_epochs_eeglab, read_events_eeglab
 from .eeglab import read_annotations_eeglab
+from .eeglab import _get_info, _check_load_mat
```

Comment:
don't expose private functions if you can

Reply:
yes, that should go. I was needing it to mock some object in the test. Private functions should not be exposed. If they are really needed by the test, then we should wrap them somehow.

Reply:
yes, that should go. I was needing it to mock some object in the test. Private functions should not be exposed. If they are really needed by the test, then we should wrap them somehow.